packages/create-plugin/templates/github/dependabot.yml

```diff
@@ -0,0 +1,50 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: weekly
+      time: "02:00"
+      timezone: "UTC"
+    open-pull-requests-limit: 5
```

Comment:
Please can we add a 5 day cooldown period here to help combat supply chain attacks on npm packages. It should exclude `@grafana/plugin-e2e` as this package needs to be kept as up to date as possible otherwise e2e tests against canary versions of Grafana might fail.